modules/EnsEMBL/Web/Component/Gene/GeneSummary.pm

```diff
@@ -256,6 +256,19 @@ sub content {
   my $type = $object->gene_type;
   $table->add_row('Gene type', $type) if $type;
 
+  if ($gene->biotype =~ /artifact/) {
+    foreach my $attr (@{$gene->get_all_Attributes}) {
+      if ($attr->code eq 'artef_dupl') {
+        my $label = 'Artefactural duplication';
```

Comment:
Can you change the label to 'Artifactual duplication' (UK spelling) ?

Reply:
The database also has the US spelling. Should we get this changed?

Reply:
We can ask for it to be changed for 108 or 109, but too late now for 107